.github/workflows/PR-3.4-U20.yaml

```diff
@@ -0,0 +1,131 @@
+name: PR:3.4 U20
+
+on:
+  pull_request:
+    branches:
+      - 3.4
+
+env:
+  EXTRA_CMAKE_OPTIONS: '-DBUILD_DOCS=ON -DPYTHON_DEFAULT_EXECUTABLE=/usr/bin/python3 -DBUILD_EXAMPLES=ON -DOPENCV_ENABLE_NONFREE=ON -DENABLE_CCACHE=OFF'
+  OPENCV_TEST_DATA_PATH: '/opencv_extra/testdata'
+  OPENCV_DOCKER_WORKDIR: '/__w/opencv/opencv'
+  PR_AUTHOR: ${{ github.event.pull_request.user.login }}
+  SOURCE_BRANCH_NAME: ${{ github.head_ref }}
+  TARGET_BRANCH_NAME: ${{ github.base_ref }}
+  ANT_HOME: '/usr/share/ant'
+  PYTHONPATH: /opencv-build/python_loader:$PYTHONPATH
+
+jobs:
+  BuildAndTest:
+    runs-on: ubuntu-20.04
+    defaults:
+      run:
+        shell: bash
+    container:
+      image: quay.io/asenyaev/opencv-ubuntu:20.04
+    steps:
+    - name: PR info
+      run: |
+        echo "PR Author: ${{ env.PR_AUTHOR }}"
+        echo "Source branch name: ${{ env.SOURCE_BRANCH_NAME }}"
+        echo "Target branch name: ${{ env.TARGET_BRANCH_NAME }}"
+    - name: Clean
+      run: find . -mindepth 1 -delete
+    - name: Fetch opencv
+      uses: actions/checkout@v3
+      with:
+        repository: opencv/opencv
+        ref: ${{ env.TARGET_BRANCH_NAME }}
+        fetch-depth: 0
+    - name: Merge opencv with ${{ env.SOURCE_BRANCH_NAME }} branch
+      run: |
+        cd ${{ env.OPENCV_DOCKER_WORKDIR }}
+        git config --global --add safe.directory ${{ env.OPENCV_DOCKER_WORKDIR }}
+        git config user.email "opencv.ci"
+        git config user.name "opencv.ci"
+        git pull -v "https://github.com/${{ env.PR_AUTHOR }}/opencv" "${{ env.SOURCE_BRANCH_NAME }}"
+    - name: Clone opencv_extra
+      run: git clone --single-branch --branch ${{ env.TARGET_BRANCH_NAME }} --depth 1 https://github.com/opencv/opencv_extra.git /opencv_extra
+    - name: Configure OpenCV
+      run: |
+        cd /opencv-build
+        cmake -G Ninja ${{ env.EXTRA_CMAKE_OPTIONS }} ${{ env.OPENCV_DOCKER_WORKDIR }}
+    - name: Build OpenCV
+      run: |
+        cd /opencv-build
+        ninja
+    - name: Accuracy
+      run: |
+        cd /opencv-build
+        accuracy_tests=($(ls bin/opencv_test*))
+        for test_name in "${accuracy_tests[@]}"; do
+          if [[ $(basename ${test_name}) != 'opencv_test_viz' ]]; then
+            xvfb-run -a ${test_name}
```

Comment:
User-wise speaking, will it be difficult to find which module reports errrors if all output of unit tests is put altogether in the 'Accuracy' section?